cfgov/v1/models/atoms.py

```diff
@@ -92,12 +93,4 @@ def clean(self, data):
         if error_dict:
             raise ValidationError("ImageBasicUrlAlt validation errors", params=error_dict)
```

Comment:
it carries additional errors resolved by default wagtail error checking in this line.
```
try:
            data = super(ImageBasicUrl, self).clean(data)
        except ValidationError as e:
            error_dict.update(e.params)
```